gradle/java.gradle

```diff
@@ -11,7 +11,7 @@ compileJava {
 }
 
 task testJar(type: Jar, dependsOn: testClasses, group: "build") {
-    appendix = 'test'
+    classifier = 'test'
```

Comment:
That is why I changed the classifier from "tests" (plural) to "test", it creates the same name now, 3rd parties should not be affected and the tests should hopefully work this time :)